internal/connector/connector.go

```diff
@@ -55,17 +55,14 @@ type ListenerOptions struct {
 func Run(ctx context.Context, options Options) error {
 	k8s, err := kubernetes.NewKubernetes()
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to create kubernetes client: %w", err)
 	}
 
-	chksm, err := k8s.Checksum()
-	if err != nil {
-		logging.Errorf("k8s checksum error: %s", err)
-		return err
-	}
+	checkSum := k8s.Checksum()
+	logging.L.Debug().Str("uniqueID", checkSum).Msg("Cluster uniqueID")
```

Comment:
I've tested this manually for now, since we don't quite yet have an easy way of testing this outside of kube.

I tested this by running `infra connector` locally using my kubectl config, and again by running it in a pod. I compared the checksum in both cases to ensure it produced the same value.